src/dataExplorer/components/Results.tsx

```diff
@@ -144,58 +124,157 @@ const Results: FC = () => {
     dupped.table.length = newLen
 
     return dupped as FluxResult['parsed']
-  }, [search, result?.parsed, view.state])
+  }, [search, result?.parsed])
 
-  let resultView
+  return (
+    <div className="data-explorer-results--view">
+      <View
+        loading={status}
+        properties={
+          {
+            type: 'simple-table',
+            showAll: false,
+          } as SimpleTableViewProperties
+        }
+        result={res}
+        timeRange={range}
+        hideTimer
+      />
+    </div>
+  )
+}
 
-  if (status === RemoteDataState.NotStarted) {
-    resultView = <EmptyResults />
-  } else {
-    if (view.state === 'table') {
-      resultView = (
-        <div className="data-explorer-results--view">
-          <View
-            loading={status}
-            properties={
-              {
-                type: 'simple-table',
-                showAll: false,
-              } as SimpleTableViewProperties
-            }
-            result={res}
-            timeRange={range}
-            hideTimer
-          />
-        </div>
-      )
-    } else {
-      resultView = (
-        <div className="data-explorer-results--view">
-          <View
-            loading={status}
-            properties={view.properties}
-            result={res}
-            timeRange={range}
-            hideTimer
-          />
-        </div>
-      )
-    }
+const GraphResults: FC = () => {
+  const {view} = useContext(ResultsViewContext)
+  const {result, status} = useContext(ChildResultsContext)
+  const {range} = useContext(PersistanceContext)
+
+  return (
+    <div className="data-explorer-results--view">
+      <View
+        loading={status}
+        properties={view.properties}
+        result={result?.parsed}
+        timeRange={range}
+        hideTimer
+      />
+    </div>
+  )
+}
+
+const WrappedOptions: FC = () => {
+  // use parent `results` so all metadata is present for the viz options
+  const {result} = useContext(ResultsContext)
+  const {setResult, setStatus} = useContext(ChildResultsContext)
+  const {view, setView /* setViewOptions*/} = useContext(ResultsViewContext)
+
+  const updateChildResults = useCallback(
+    update => {
+      setView({
+        ...view,
+        properties: {
+          ...view.properties,
+          ...update,
+        },
+      })
+    },
+    [setStatus, setResult]
+  )
+
+  // TODO: make component, for QxBuilder-specific graph subquery options
+  const subQueryOptions = null
+
+  return (
+    <>
+      {subQueryOptions}
+      {/* <SubQueryOptions update={setViewOptions} /> */}
```

Comment:
This `SubQueryOptions` component, or a functional equivalent, will be done in followup PRs. a.k.a. other Epic subtasks.

Reply:
Since this is still production code, I'd say either remove the commented lines, or create a placeholder `SubQueryOptions` component that doesn't have any functionality in it similar to how you're doing `subQueryOptions = null`